CodeAssignment_MVVM_Combine/Sources/Controllers/HomeViewController.swift

```diff
@@ -17,63 +18,76 @@ class HomeViewController: UIViewController {
         button.titleLabel?.font = UIFont.boldSystemFont(ofSize: 14)
         button.setTitle("GET", for: .normal)
         button.setTitleColor(UIColor.systemBlue, for: .normal)
-        /// Snapkitライブラリで既にtranslatesAutoresizingMaskIntoConstraintsをfalseにしているので、記載不要
+        // Snapkitライブラリで既にtranslatesAutoresizingMaskIntoConstraintsをfalseにしているので、記載不要
         button.clipsToBounds = true
         button.layer.cornerRadius = 8
         button.layer.borderWidth = 1
         button.layer.borderColor = UIColor.systemGray.cgColor
-        /// addActionを使うことで、objc funcを使わなくても済む
-        /// UIActionのclosureを用いて関数を呼び出す
-        button.addAction(UIAction { [weak self] _ in
+        // addActionを使うことで、objc funcを使わなくても済む
+        // UIActionのclosureを用いて関数を呼び出す
+        button.addAction(.init { [weak self] _ in
             self?.didTapGetAPIButton()
         }, for: .touchUpInside)
         
         return button
     }()
     
-
+    /// ViewModel
+    private let viewModel = HomeViewModel()
+    /** storeでAnyCancellableを保存しておいて、当該の変数がdeinitされるとき、subscribeをキャンセルする方法
+    - Setで複数のSubscription（購読）を１つにまとめることができ、Subscriptionの値を保持する
+     */
+    private var cancellables = Set<AnyCancellable>()
+    private var repositoriesForView: Repositories?
```

Comment:
ここも `ForView` は不要ですね！
```suggestion
    private var repositories: Repositories?
```